src/main/groovy/de/dkfz/eilslabs/batcheuphoria/TestExecutionService.groovy

```diff
@@ -7,6 +7,8 @@
 package de.dkfz.eilslabs.batcheuphoria
```

Comment:
IMPORTANT: You are missing the copyright headers in all your files! Please take a look at the Roddy project to see how this is done. It is called copyright notice or so in the configuration.